app/helpers/accordion_helper.rb

```diff
@@ -1,5 +1,5 @@
 module AccordionHelper
-  # Options hash values: hide_header, hide_close_link
+  # Options hash values: cntnr_classes, hide_header, hide_close_link
```

Comment:
A question, why `cntnr_classes`? I'm not certain what cntnr means in this context. I actually thought it meant 'centerer', and referred to the modal?